crates/node/src/rpc_client/mod.rs

```diff
@@ -9,17 +9,46 @@ use jsonrpsee::{
     http_client::{HttpClient, HttpClientBuilder},
 };
 use std::error::Error;
+use std::time::Duration;
+
+/// A RPC client builder for connecting to the Gevulot network
+#[derive(Debug)]
+pub struct RpcClientBilder {
```

Comment:
```suggestion
pub struct RpcClientBuilder {
```
Small typo here. Please fix it across the whole PR. :slightly_smiling_face:

Reply:
fixed.